### ISSUE TITLE
V8: Make the save-options available by keyboard

### ISSUE THREAD

**Issue by nielslyngsoe** [umbraco/Umbraco-CMS#5443]
These options are not available by keyboard, I would love them to be.

![image](https://user-images.githubusercontent.com/6791648/57517104-63a8ea80-7316-11e9-8845-e2c0821006da.png)

**Comment by electricsheep** [comment]
@nielslyngsoe I've created a PR with hotkeys for "Schedule" and "Publish with descendants" (there was an existing shortcut for "Unpublish". 

Any preference on the exact shortcuts to use? I've gone with alt+shift+s for "Schedule" and alt+shift+p for "Publish with descendants"

**Comment by nielslyngsoe** [comment]
@electricsheep My issue was meant to say that the menu cant be opened by using the keyboard, and thereby the options cant be accessed by keyboard. But having keyboard shortcuts for the options is also super great 

But my though was that you should be able to tab the little arrow next to "Save and publish", and when its in focus you can press the Enter-key to open the menu and tab through the options.

let me know what you think about that :-)

**Comment by electricsheep** [comment]
@nielslyngsoe Whoops ok, that makes sense. I have created a new PR so the menu can be tabbed to and triggered.

By the way, I noticed another bug while testing this fix - https://github.com/umbraco/Umbraco-CMS/issues/5478
